connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -247,6 +248,13 @@ public boolean isSinkConnector(String connName) {
         }
     }
 
+    /*
+     * @return the ConnectorType for the given connector name
```

Comment:
Please add a sentence describing the method. And, this method, like isSinkConnector, can throw a ConnectException, so that should be added to the JavaDoc as well.